presidio-anonymizer/presidio_anonymizer/entities/analyzer_result.py

```diff
@@ -0,0 +1,101 @@
+"""
+AnalyzerResult is the exact copy of the recognizer result.
```

Comment:
I don't think this line is required.

Reply:
Sadly, the linter thinks otherwise.

Reply:
Wdym? The whole comment - where does it belong to?
I think you can just remove the whole comment block